boards/st/stm32n6570_dk/stm32n6570_dk.dts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/n6/stm32n657X0.dtsi>
+#include <st/n6/stm32n657x0hxq-pinctrl.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "STMicroelectronics STM32N6570 Discovery Kit board";
+	compatible = "st,stm32n6570-dk";
+
+	chosen {
+		zephyr,console = &usart1;
+		zephyr,shell-uart = &usart1;
+		zephyr,sram = &axisram2;
+	};
+
+	aliases {
+		led0 = &green_led_1;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+		green_led_1: led_1 {
```

Comment:
Still need an empty line above this line and line 32.

Reply:
Done